uPortal-security/uPortal-security-core/src/main/java/org/apereo/portal/security/firewall/README.md

```diff
@@ -1,15 +1,15 @@
 RequestParameterPolicyEnforcementFilter
 =======================================
 
-The `RequestParameterPolicyEnforcementFilter` is a blunt generic Java Servlet Filter suitable for 
+The `RequestParameterPolicyEnforcementFilter` is a blunt generic Java Servlet Filter suitable for
 
  * in general, blocking malicious requests involving unexpectedly duplicate request parameters or request parameters with unexpected and poorly handled characters in their values.
- * in specific, blocking the malicious requests involved in `CVE-2014-4172`, to which certain Java CAS Client versions 
+ * in specific, blocking the malicious requests involved in `CVE-2014-4172`, to which certain Java CAS Client versions
 are vulnerable, including Java CAS Client versions that have been included in uPortal releases.
 
-It is a fork of that offered to the `cas-server-security-filter` project at https://github.com/Jasig/cas-server-security-filter/pull/6 .
+It is a fork of that offered to the `cas-server-security-filter` project at https://github.com/uPortal-Project/cas-server-security-filter/pull/6 .
```

Comment:
This repo is still in the Jasig Github org.

Reply:
Thanks Jonathan, I fixed it with apereo link (removing the redirect from Jasig)